tap_snapchat_ads/streams.py

```diff
@@ -41,6 +41,17 @@ def update_currently_syncing(state, stream_name):
         singer.set_currently_syncing(state, stream_name)
     singer.write_state(state)
 
+def get_hourly_stats_fields():
+    """
+    removes fields [attachment_frequency,attachment_uniques,frequency,uniques] for AdSquadStatsHourly, AdStatsHourly,
+     CampaignStatsHourly  from ALL_STATS_FIELDS
+    """
+    unwanted_fields = ['attachment_frequency','attachment_uniques','frequency','uniques']
+    all_fields = ALL_STATS_FIELDS
```

Comment:
Should we consider using deepcopy()?
Because `all_fields` and `ALL_STATS_FIELDS` are pointing to the same memory location.

Reply:
addressed and added unit test case